packages/constellation/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -1,26 +1,33 @@
 /* eslint-disable import/no-unresolved */
 import '../styles.css'
 
-import React, { CSSProperties, useMemo } from 'react'
+import React, { useEffect } from 'react'
 
+import { DarkModeContext } from './themeContext'
 import { ThemeProviderComponent } from './ThemeProvider.types'
 
 const ThemeProvider: ThemeProviderComponent = ({ children, theme, colorOverride = {} }) => {
-  const colorOverrideStyles = useMemo((): CSSProperties => {
-    return Object.keys(colorOverride).reduce(
-      (newColorOverride, color) => ({
-        ...newColorOverride,
-        [`--color-${color}`]: colorOverride[color as keyof typeof colorOverride],
-      }),
-      {},
+  useEffect(() => {
+    if (theme === 'dark') {
+      document.body.classList.add('theme-dark')
+      document.body.classList.remove('theme-light')
+    } else {
+      document.body.classList.add('theme-light')
+      document.body.classList.remove('theme-dark')
+    }
```

Comment:
I think this is ok for now as we're not quite building for production, but I think it's definitely something we'll want to run by the Blockchain team.

We'll want to make a case for why we need to do this, the alternatives to this, and the potential risks and how to avoid them (mostly covered in the ADR).

I can also see this become a bit unwieldy when adding new themes.